src/main/kotlin/racingcar/domain/Cars.kt

```diff
@@ -19,5 +19,9 @@ class Cars(
         fun of(carCount: Int): Cars {
             return Cars((0 until carCount).map { Car() })
         }
+
+        fun of(carNames: List<CarName>): Cars {
+            return Cars(carNames.map { Car(it) })
```

Comment:
```suggestion
            return Cars(carNames.map(::Car))
```
전달하는 인자의 타입 및 개수와 일치하는 경우에는, 메서드 참조를 활용할 수 있어요.
메서드 참조를 활용하면 전달받은 인자에 어떠한 변경도 없이 바로 전달한다는 것을 보장할 수 있어요.

Reply:
오 코틀린도 메서드 참조가 있었군요! 감사합니다 😄